openbb_terminal/stocks/fundamental_analysis/fa_controller.py

```diff
@@ -589,16 +608,33 @@ def call_balance(self, other_args: List[str]):
             dest="b_quarter",
             help="Quarter fundamental data flag.",
         )
+        parser.add_argument(
+            "-s",
+            "--source",
+            help="Source to get fundamentals from",
+            default="av",
```

Comment:
same here

Reply:
Read the text of the pr :P

Reply:
Gotcha. Completely ignored that wall of text 🤣